faculty/clients/experiment.py

```diff
@@ -195,3 +230,65 @@ def get_run(self, project_id, run_id):
         """
         endpoint = "/project/{}/run/{}".format(project_id, run_id)
         return self._get(endpoint, ExperimentRunSchema())
+
+    def list_runs(
+        self,
+        project_id,
+        experiment_ids=None,
+        lifecycle_stage=None,
+        start=None,
+        limit=None,
+    ):
+        """List experiment runs.
+
+        This method returns pages of runs. If less than the full number of runs
+        for the job is returned, the ``next`` page of the returned response
+        object will not be ``None``:
+
+        >>> response = client.list_runs(project_id)
+        >>> response.pagination.next
+        Page(start=10, limit=10)
+
+        Get all experiment runs by making successive calls to ``list_runs``,
+        passing the ``start`` and ``limit`` of the ``next`` page each time
+        until ``next`` is returned as ``None``.
+
+        Parameters
+        ----------
+        project_id : uuid.UUID
+        experiment_ids : List[int], optional
+            To filter runs of experiments with the given IDs only. If an empty
+            list is passed, a result with an empty list of runs is returned.
```

Comment:
Probably worth adding something like:
"By default, all experiments are returned"

Reply:
I fixed the commend.

Reply:
OK, let's factor out the pagination model in a separate PR.